### PR TITLE
studio: update scene item UI

### PR DESCRIPTION
**Pull request by achen5671**
## Summary
https://tsu.atlassian.net/browse/IR-3315

## References
closes #_insert number here_

### CODE REVIEW

File: packages/ui/src/components/editor/panels/Scenes/container/index.tsx [hunk 1]
```diff
@@ -108,28 +109,40 @@ export default function ScenesPanel() {
...
+                  <div className="inline-flex items-start justify-between self-stretch px-1">
+                    <div className="inline-flex w-full flex-col items-start justify-start">
+                      <Text className="truncate text-sm leading-5 dark:text-[#A3A3A3]">{getSceneName(scene)}</Text>
+                      <Text
+                        component="h3"
+                        fontSize="xs"
+                        fontWeight="light"
+                        className="h-3.5 w-40 font-['Figtree'] leading-5 text-neutral-100"
+                      >
+                        {t('multitenancy:scene.edited')}{' '}
```
Comment (HexaField) [inline packages/ui/src/components/editor/panels/Scenes/container/index.tsx]: core engine i18n needs to go in the core engine files

**Review by HexaField — CHANGES_REQUESTED** [review]

File: packages/ui/src/components/editor/panels/Scenes/container/index.tsx [hunk 2]
```diff
@@ -108,28 +109,40 @@ export default function ScenesPanel() {
...
+                        {t('common:timeAgo', { time: timeAgo(new Date(scene.updatedAt)) })}
```
Comment (HexaField) [inline packages/ui/src/components/editor/panels/Scenes/container/index.tsx]: for some reason it seems like updatedAt is returning only a few seconds ago
Screenshot from 2024-07-23 12-24-13

**Review by HexaField — CHANGES_REQUESTED** [review]

File: packages/ui/src/components/editor/panels/Scenes/container/index.tsx [hunk 3]
```diff
@@ -108,28 +109,40 @@ export default function ScenesPanel() {
...
                     crossOrigin="anonymous"
-                    className="block h-full grow cursor-pointer self-center rounded-t-lg object-cover"
+                    className="object-fit block h-44 w-full grow cursor-pointer self-center rounded-lg"
                     onClick={() => onClickScene(scene)}
                   />
-                  <div className="flex items-center justify-between px-4 py-1">
-                    <Text className="truncate text-sm leading-5 dark:text-[#A3A3A3]">{getSceneName(scene)}</Text>
+                  <div className="inline-flex items-start justify-between self-stretch px-1">
+                    <div className="inline-flex w-full flex-col items-start justify-start">
+                      <Text className="truncate text-sm leading-5 dark:text-[#A3A3A3]">{getSceneName(scene)}</Text>
```
Comment (aditya-mitra) [inline packages/ui/src/components/editor/panels/Scenes/container/index.tsx]:
```suggestion
                      <Text className="truncate text-sm text-[#A3A3A3]">{getSceneName(scene)}</Text>
```

`text-sm` is leading-5 by default
also studio is in dark mode

File: packages/ui/src/components/editor/panels/Scenes/container/index.tsx [hunk 4]
```diff
@@ -108,28 +109,40 @@ export default function ScenesPanel() {
...
                       e.currentTarget.src = 'static/ir.svg'
                     }}
                     crossOrigin="anonymous"
-                    className="block h-full grow cursor-pointer self-center rounded-t-lg object-cover"
+                    className="object-fit block h-44 w-full grow cursor-pointer self-center rounded-lg"
                     onClick={() => onClickScene(scene)}
                   />
-                  <div className="flex items-center justify-between px-4 py-1">
-                    <Text className="truncate text-sm leading-5 dark:text-[#A3A3A3]">{getSceneName(scene)}</Text>
+                  <div className="inline-flex items-start justify-between self-stretch px-1">
```
Comment (aditya-mitra) [inline packages/ui/src/components/editor/panels/Scenes/container/index.tsx]:
```suggestion
                  <div className="inline-flex justify-between self-stretch px-1">
```

`items-start` is default

File: packages/ui/src/components/editor/panels/Scenes/container/index.tsx [hunk 5]
```diff
@@ -108,28 +109,40 @@ export default function ScenesPanel() {
               {scenes.map((scene) => (
                 <div
                   key={scene.id}
-                  className="my-2 flex h-[240px] w-[250px] flex-col justify-end rounded-lg bg-theme-surface-main"
+                  className="col-span-2 inline-flex h-64 w-64 cursor-pointer flex-col items-start justify-start gap-2 rounded-lg bg-[#191B1F] p-3 lg:col-span-1"
```
Comment (aditya-mitra) [inline packages/ui/src/components/editor/panels/Scenes/container/index.tsx]:
```suggestion
                  className="col-span-2 inline-flex h-64 w-64 cursor-pointer flex-col gap-2 rounded-lg bg-[#191B1F] p-3 lg:col-span-1"
```

File: packages/ui/src/components/editor/panels/Scenes/container/index.tsx [hunk 6]
```diff
@@ -108,28 +109,40 @@ export default function ScenesPanel() {
...
-                    <Text className="truncate text-sm leading-5 dark:text-[#A3A3A3]">{getSceneName(scene)}</Text>
+                  <div className="inline-flex items-start justify-between self-stretch px-1">
+                    <div className="inline-flex w-full flex-col items-start justify-start">
+                      <Text className="truncate text-sm leading-5 dark:text-[#A3A3A3]">{getSceneName(scene)}</Text>
+                      <Text
+                        component="h3"
+                        fontSize="xs"
+                        fontWeight="light"
+                        className="h-3.5 w-40 font-['Figtree'] leading-5 text-neutral-100"
+                      >
```
Comment (aditya-mitra) [inline packages/ui/src/components/editor/panels/Scenes/container/index.tsx]:
```suggestion
                      <Text
                        fontSize="xs"
                        fontWeight="light"
                        className="h-3.5 w-40 leading-5 text-neutral-100"
                      >
```

we have figtree as the default font everywhere

**Review by aditya-mitra — CHANGES_REQUESTED** [review]

**Comment by hanzlamateen** [comment]
@HexaField since this and the one in console appears same.

Why dont we just make a control common across both implementations?

https://github.com/theinfinitereality/eepro-multitenancy/blob/c19917f033755b4e8b6bfef8160f504aa35a8477/components/pages/MyProjects/SceneItem.tsx

**Comment by HexaField** [comment]
> @HexaField since this and the one in console appears same.
> 
> Why dont we just make this control common across both implementations?
> 
> https://github.com/theinfinitereality/eepro-multitenancy/blob/c19917f033755b4e8b6bfef8160f504aa35a8477/components/pages/MyProjects/SceneItem.tsx

That's a great point, yes let's do that

File: packages/ui/src/components/editor/panels/Scenes/container/index.tsx (hunk 2 above)
Comment (achen5671) [reply packages/ui/src/components/editor/panels/Scenes/container/index.tsx]: weird, I don't see that

Comment (achen5671) [reply packages/ui/src/components/editor/panels/Scenes/container/index.tsx]: Screenshot 2024-07-25 at 1 14 09 PM

**Comment by achen5671** [comment]
related pr in mt repo: https://github.com/theinfinitereality/eepro-multitenancy/pull/364

**Review by hanzlamateen — APPROVED** [review]

**Comment by SYBIOTE** [comment]
this PR causes the scene menu modal to be cut off

**Comment by SYBIOTE** [comment]
[#10790](https://github.com/EtherealEngine/etherealengine/pull/10790)

please refer to this PR

**Pull request merged**